libraries/ESP8266WiFi/src/include/ClientContext.h

```diff
@@ -333,6 +333,7 @@ class ClientContext
         return _written;
     }
 
+#define TCP_WRITE_CHUNK_SIZE 256
```

Comment:
Can we change this to a ClientContext member variable? Then we can later add an API to WiFiClient to configure the chunk size at runtime. This variable can be default-initalized to 256 for now.